Library/Formula/anjuta.rb

```diff
@@ -0,0 +1,159 @@
+class Anjuta < Formula
+  desc "GNOME Integrated Development Environment"
+  homepage "http://anjuta.org"
+  url "https://download.gnome.org/sources/anjuta/3.16/anjuta-3.16.0.tar.xz"
+  sha256 "77432a01b37b46f5558bf4cc435f9a023abf17c5273d472e17dff1f9b3d9943c"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "itstool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gtk-doc" => :build
+  depends_on "yelp-tools" => :build
+  depends_on "gnome-common" => :build
+  depends_on "bison" => :build
+  depends_on "gtksourceview3"
+  depends_on "libxml2"
+  depends_on "libgda"
+  depends_on "gdl"
+  depends_on "vte3"
+  depends_on "hicolor-icon-theme"
+  depends_on "shared-mime-info"
+  depends_on :python
```

Comment:
Needs to be either `depends_on :python if MacOS.version <= :snow_leopard` or `depends_on "python"`. Try for the former, we've been trying to eradicate the latter.